libfreerdp/crypto/test/TestFuzzCryptoBerReadLenght.c

```diff
@@ -0,0 +1,8 @@
+#include <freerdp/crypto/crypto.h>
+
+extern int LLVMFuzzerTestOneInput(const uint8_t* Data, size_t Size)
+{
+	ber_read_length((wStream*)Data, &Size);
```

Comment:
this is really wrong, you can not simply cast any buffer to a `wStream` and expect it to not crash